databricks_cli/sdk/service.py

```diff
@@ -1058,6 +1058,14 @@ def run(self, pipeline_id=None, headers=None):
         return self.client.perform_query('POST', '/pipelines/{pipeline_id}/run'.format(pipeline_id=pipeline_id),
                                          data=_data, headers=headers)
 
+    def start(self, pipeline_id=None, full_refresh=False, headers=None):
+        _data = {}
+        if full_refresh:
+            _data['full_refresh'] = full_refresh
+        _data['cause'] = 'API_CALL'
```

Comment:
@calving-db this is now `USER_ACTION`, correct?

Reply:
Yes

Reply:
updated to USER_ACTION, do we need to be careful about compatibility issue between cli and the databricks backend version (since this is a new enum that we are introducing)?